src/main/java/org/reactome/release/qa/graph/SingleAttributeSoleValueCheck.java

```diff
@@ -31,6 +32,11 @@ protected boolean isEscaped(GKInstance inst, String attName) throws Exception {
         if (super.isEscaped(inst, attName)) {
             return true;
         }
+        // Check if instance DbId is in skiplist.
+        if (super.inSkipList(inst, skiplist)) {
```

Comment:
Just curious why there is a call to super here when there isn't one in the other classes calling `inSkipList`?

Reply:
The skiplists were requested _en masse_ from Lisa based on a lot of noisy weekly QA results being returned. So this check wasn't actually created by me, but the methods utilized the super approach (compare the `isEscaped` method usage in different classes for examples) and so I did the same so as to keep the code style consistent

Reply:
Ah okay - I think “super” is required for “isEscaped” here because it is calling the implementation in the parent class (without “super”, this method would call itself and there would be infinite recursion), but “super” is optional for “inSkipList” because the method doesn’t exist in this class (i.e. it’s not overridden) so it knows to go to the parent class with or without “super”.  It might be better to remove “super” here, because it seems to imply the method exists in this and the  parent class.

Reply:
More on `super`: https://docs.oracle.com/javase/tutorial/java/IandI/super.html

Reply:
The new implementation with the `SkipList` class removed the `super` implementation